broker/config/settings.yml

```diff
@@ -402,6 +402,17 @@ defaults: &defaults
     url: https://api.bosh-lite.com
     username: admin
     password: admin
+  
+  #############################
+  # CREDHUB SETTINGS #
+  #############################
+  cred_provider:
+    credhub_url: https://10.0.3.11:8443/api/
+    credhub_uaa_url: https://10.0.3.11:8443
+    credhub_client_id:  credhub-admin
```

Comment:
Shouldn't the UAA related configs be rename as uaa_ instead of credhub_

Reply:
Answered in BnR [PR](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/35#discussion_r196321935)